client/van_connector_create_test.go

```diff
@@ -0,0 +1,115 @@
+package client
+
+import (
+	"context"
+	"os"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/skupperproject/skupper/api/types"
+	"gotest.tools/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
+)
+
+func TestConnectorCreateError(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
for now van_connector_create is not using ctx, this "cancel", call is for a future change? or just in case ... right?

Reply:
Yes, use of ctx is intended for a future change. The client calls should be utilizing but does not presently.